doc/lua_api.txt

```diff
@@ -971,20 +971,64 @@ There are a bunch of different looking node types.
 Look for examples in `games/minimal` or `games/minetest_game`.
 
 * `normal`
+    * A node-sized cube.
 * `airlike`
+    * Invisible, uses no texture.
 * `liquid`
+    * The cubic source node for a liquid.
 * `flowingliquid`
+    * The flowing version of a liquid, appearing with various heights and
+      slopes.
 * `glasslike`
+    * Often used for partially-transparent nodes.
+    * Only external sides of textures are visible.
 * `glasslike_framed`
+    * All face-connected nodes are drawn as one volume within a surrounding
+      frame.
+    * The frame appearence is generated from the edge pixels of the first
+      texture specified in `tiles` (preferably 16x16 pixels).
+    * The glass 'shine' (or other desired detail) on each node face is supplied
+      by the second texture specified in `tiles`.
 * `glasslike_framed_optional`
+    * This switches between the above 2 drawtypes according to the clientside
+      setting 'connected glass'.
 * `allfaces`
+    * Often used for partially-transparent nodes.
+    * External and internal sides of textures are visible.
 * `allfaces_optional`
+    * Often used for leaves nodes.
+    * This switches between `normal`, `glasslike` and `allfaces` according to
+      the clientside leaves setting: Opaque / Simple / Fancy.
+    * With 'Simple leaves' selected, the texture specified in `special_tiles`
+      is used instead, if present. This allows a visually thicker texture to be
+      used to compensate for how `glasslike` reduces visual thickness. 
 * `torchlike`
+    * A single vertical texture.
+    * If placed on top of a node, uses the first texture specified in `tiles`.
+    * If placed against the underside of a node, uses the second texture
+      specified in `tiles`.
+    * If placed on the side of a node, uses the third texture specified in
+      `tiles`, and is perpendicular to that node.
 * `signlike`
+    * A single texture parallel to, and mounted against, the top, underside or
+      side of a node.
 * `plantlike`
+    * Two vertical and diagonal textures at right-angles to each other.
+    * See `paramtype2 == "meshoptions"` above for other options.
 * `firelike`
+    * Appears in different ways as textures placed against the faces of
```

Comment:
"different ways" is very vague, and from reading it alone it's impossible to understand what it means. Can this be more precise?

Reply:
It can.